nexus/db-queries/src/db/raw_query_builder.rs

```diff
@@ -135,7 +135,7 @@ impl QueryBuilder {
         Value: diesel::serialize::ToSql<BindSt, Pg> + Send + 'static,
         BindSt: Send + 'static,
     {
-        self.query = self.query.take().map(|q| q.bind(b));
+        self.query = self.query.take().map(|q| q.bind::<BindSt, Value>(b));
```

Comment:
Oh interesting, was this necessary?

Reply:
no, this was a left-over for me debugging PEBKAC! reverted in 35cffce2e